### PR TITLE
documentation: ZENKO-1357 Fix_PDF_Output

### PR DESCRIPTION
**Pull request by wabernat**
This commit makes documentation build to PDF using the Scality
template for the latexpdf toolchain. This does not get us up to
production quality, but gets us much closer.

Remaining issues:
* Table formatting (ZENKO-1417)
* Project organization (This produces a good help website, searchable
  across all "books". The PDF contains all three "books" (inst, ops,
  and ref) as one hefty PDF. (ZENKO-1407)

Additionally, I modified the table in
Installation/topics/adding_NFS_storage.rst in anticipation of
ZENKO-1417.

### CODE REVIEW

**Comment by wabernat** [comment]
@bert-e reset

**Comment by wabernat** [comment]
@bert-e approve

**Review by ssalaues — CHANGES_REQUESTED** [review]
Please delete all the `*.pyc` files that have been included. These are the compiled byte code files that shouldn't be part of the commit. Also would be a good idea to add `*.pyc` to the .gitignore file to prevent this in the future.

**Comment by wabernat** [comment]
Made requested changes.

**Review by ssalaues — APPROVED** [review]

File: docs/scaldoc/screenshot/__init__.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Black = Color.Black
+Red = Color.Red
+Green = Color.Green
+Yellow = Color.Yellow
+Blue = Color.Blue
+Magenta = Color.Magenta
+Cyan = Color.Cyan
+White = Color.White
+Invert = Color.Invert_bgb
+Invert = Color.Invert_bgw
```
Comment (jonathan-gramain) [inline docs/scaldoc/screenshot/__init__.py:18]: This seems to override the previous line

**Review by jonathan-gramain — APPROVED** [review]

**Pull request merged**